recipes/elector/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+mkdir -p $PREFIX/bin/
+# build and install binary tools
+cd src/simulator
+sed -i "s#CC=g++#CC=${CXX}#" makefile
+make
+cp simulator $PREFIX/bin/
+
+cd ../poa-graph
+sed -i "s#CC = gcc#CC=${CC}#" Makefile
+make poa
+cp poa $PREFIX/bin/
+
+cd ../utils
+sed -i "s#CC=g++#CC=${CXX}#" makefile
+make
+cp fq2fa $PREFIX/bin/
+
+cd ../split
+sed -i "s#CC=g++#CC=${CXX}#" makefile
```

Comment:
Interesting, looks like you change is correct, meaning it's wrong in the source.
In this case, just use `make CC="${CXX}" CXX="${CXX}"`.